matlab/README.md

```diff
@@ -27,19 +27,20 @@ To build the Docker image, follow these steps:
    FROM matlab AS matlab-install-stage
    ```
 
-3. If MATLAB is not installed at the location `/usr/local/MATLAB` in the image built in step 1, edit the following line to point to you MATLAB installation:
+3. MATLAB should be on the path in the image built used in step 1.
```

Comment:
Could we change "image built used" to "image used"?
Doc guidelines recommend to avoid "should" could we change this to:

MATLAB must be on the system path in the image built used in step 1.
If it is not, then edit the following line to point to your MATLAB installation: